cartoframes/data/observatory/catalog/subscriptions.py

```diff
@@ -3,6 +3,11 @@
 
 
 class Subscriptions(object):
```

Comment:
This class is not exposed in the reference either

Reply:
I'm going to expose it 👍